src/main.rs

```diff
@@ -7,6 +7,7 @@ mod game_session;
 use game_session::{GameSession, SessionResult, print_game_board};
 use ask::ask_yes_no;
 
+// REVIEW: style: not important, but I personally prefer clearer
```

Comment:
Oops. 

...clearer error messages then just a default panic printer

Reply:
It's `Debug` printing, not panic printer. Still needs to be addressed, tough.